pprof.go

```diff
@@ -0,0 +1,34 @@
+package conf
+
+import "runtime"
+
+// PPROF is a confiuration struct which can be used to configure the runtime
+// profilers of programs.
+//
+//	config := struct{
+//		PPROF `conf:"pprof"`
+//	}{
+//		PPROF: conf.DefaultPPROF(),
+//	}
+//	conf.Load(&config)
+//	conf.SetPPROF(config.PPROF)
+//
+type PPROF struct {
+	BlockProfileRate     int `conf:"block-profile-rate"     help:"Sets the mutex profile fraction to enable runtime profiling of lock contention, zero disables mutex profiling" validate:"min=0,max=1"`
```

Comment:
help text looks copied.

Reply:
I do this to check if you read the diff :D ... 

No for real I just had coffee, still waking up here.